drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -48,24 +48,22 @@ def run!
     valid_containers.values.each do |processor|
       processor.new(self).information_date(custom_form.assessment.assessment_date) if custom_form.assessment.present?
     end
-  end
 
-  def save_enrollment
-    enrollment_factory.save! if enrollment_factory.present?
+    @owner.enrollment = enrollment_factory if @owner.is_a?(Hmis::Hud::CustomAssessment)
```

Comment:
@gigxz fix.. can't actually do this because of WIP enrollment table. This effectively moves the assessment out of WIP status, which we don't want.